cypress/integration/statusSpec.js

```diff
@@ -1,10 +1,7 @@
-import { testResponseCode, testContentType } from '../support/metaTestHelper';
-
-describe('Simorgh Status', () => {
-  it('should return 200', () => {
-    testResponseCode('/status', 200);
-  });
+import { testContentType } from '../support/metaTestHelper';
+import describeForLocalOnly from '../support/describeForLocalOnly';
 
+describeForLocalOnly('Service worker', () => {
```

Comment:
Why is this test changed to only work on the local app? We should check that these files exist and are available:
https://www.bbc.co.uk/news/articles/sw.js 
https://www.test.bbc.co.uk/news/articles/sw.js

Reply:
They exist but don't have the content-type, I could change it to be a `200` status check

Reply:
Ahhh I misunderstood @dr3 before, it doesn't return `charset=UTF-8` on the content-type. I'll update the tests to reflect this